demos/common/python/openvino/model_zoo/model_api/models/hpe_associative_embedding.py

```diff
@@ -17,13 +17,18 @@
 import numpy as np
 from scipy.optimize import linear_sum_assignment
 
+
+
 from .image_model import ImageModel
+from .types import NumericalValue, StringValue
 from .utils import resize_image
 
 
 class HpeAssociativeEmbedding(ImageModel):
-    def __init__(self, model_adapter, target_size, aspect_ratio, prob_threshold, delta=0.0, size_divisor=32, padding_mode='right_bottom'):
-        super().__init__(model_adapter)
+    __model__ = 'HPE-assosiative-embeddings'
```

Comment:
```suggestion
    __model__ = 'HPE-assosiative-embedding'
```
Also, the same in human_pose_estimation_demo

Reply:
done